driver/src/test/scala/Common.scala

```diff
@@ -167,4 +166,45 @@ object Common {
 
     slowProxy.stop()
   }
+
+}
+
+trait CommonAuth {
```

Comment:
I would rather keep in `Common`

Reply:
There are 2 `Common` objects, one for most tests and one for the jmx stuff and both required the same handling to enable X509 client auth.

Reply:
The test classes are separate

Reply:
In order to run both standard and jmx tests with and without x509 auth I need to modify the connection. Logic to do it is similar so I put it in a separate trait that I use from both places. The trait has only auth related functionality. Can you please suggest a better way?